docs/en/05_Contributing/02_Request_for_comment.md

```diff
@@ -0,0 +1,52 @@
+# Request for comment (RFC)
+
+## Why RFCs?
+This is part of the SilverStripe core decision-making process and addresses the gap between the idea phase and the pull request submission (and merge).
+
+The rationale behind this process is to:
+ * Encourage visibility on decision making
+ * Clarity on what the proposal is, its rationale and impact
+ * Reduce unnecessary work when coding large features is done without community and core-committers buy-in
+ * Improved likelihood of an optimal solution being merged
+
+The important thing to understand about the RFCs is that these are NOT a way to request features. Rather, they are a way to bring clarity so people understand what change is being proposed.
+
+## When to write an RFC?
+
+We intend RFCs to be the primary mechanisms for proposing major new features, for collecting community input on an issue, and for documenting the design decisions that have gone into SilverStripe. The RFC author is responsible for building consensus within the community and documenting dissenting opinions.
+
+Before writing the actual summary RFC document, the idea should have already had a wide range of discussion in various community communication channels. Once discussions reach a point where you think most of the difficulties have been worked through then create the RFC using the template provided.
+
+The benefits of writing an RFC for non-trivial feature proposals are:
+ * Obtaining a preliminary approval from core-committers on an architecture before code is completed, to mitigate the risk of a non-merge after a PR is submitted
+ * Community becomes aware of incoming changes prior to the implementation
+ * RFC can be used as a basis for documentation of the feature
+	
+## How to write an RFC?
+### Template
+The following heading can act as a template to starting your RFC.
+ * **Introduction** - include a reference #, title, author
+ * **Metadata** - standardised header containing at least the Author(s), Status and Version fields.
```

Comment:
Should probably outline what the valid values are for *Status* and their progression e.g. Pending review -> Approved... anything else?

Also the Version field is a bit confusing, especially if you think in SemVer terms. Probably should increase the version number by 1 every time it's edited (whole numbers only, no point releases). Think a Last Updated field should accompany this field since there's no version history or Edited date in GitHub issues and comments.